MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-recursive-include django_downloadview *
+recursive-include demo *.py *.rst *.html *.json *.txt
```

Comment:
I would use it for the tests and possibly provide it as an example inside the Debian package.